src/psyclone/tests/test_files/dynamo0p3/infrastructure/function_space/function_space_mod.F90

```diff
@@ -1305,7 +1305,7 @@ end function get_stencil_dofmap
 !> @return map The stencil_dofmap object to return
 function get_stencil_2D_dofmap(self, stencil_shape, stencil_extent) result(map)
 
-  use stencil_2D_dofmap_mod, only: stencil_2D_dofmap_type
+  use stencil_2d_dofmap_mod, only: stencil_2D_dofmap_type
```

Comment:
LFRic has renamed the corresponding file, so in order to be consistent (and trigger an error early in case that we have an explicit reference to that file in some time in the future) I've also renamed this file.